rootfs/api/settings/testing.py

```diff
@@ -37,3 +38,5 @@
 # How long k8s waits for a pod to finish work after a SIGTERM before sending SIGKILL
 KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS = int(os.environ.get('KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS', 2))  # noqa
 KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC = '{"spec":{"hard":{"pods":"10"}}}'
+
+DEIS_DEFAULT_CONFIG_TAGS = str(os.environ.get('DEIS_DEFAULT_CONFIG_TAGS', ''))
```

Comment:
I'm noticing how here the value is cast to string w/ `str()` but not in `rootfs/api/settings/production.py` -- is there a reason?